infinite_tracing/test/infinite_tracing/channel_test.rb

```diff
@@ -4,6 +4,7 @@
 # frozen_string_literal: true
 
 require_relative '../test_helper'
+SimpleCov.command_name "test:multiverse[infinite_tracing]" if RUBY_VERSION >= '2.7.0'
```

Comment:
Could infinite tracing also use the `SimpleCovHelper`?

Reply:
Definitely looks like it :)